### ISSUE TITLE
JOIN USING triggers a query anti-pattern

### ISSUE THREAD

**Issue by perdiesman** [jarulraj/sqlcheck#61]
When doing a join with a using such as:
```sql
SELECT t1.col1, t2.col2
FROM t1
    JOIN t2 USING (col1);
```
this is triggering:
```
(HIGH RISK) (QUERY ANTI-PATTERN) JOIN Without Equality Check
● Use = with JOIN:
JOIN should always have an equality check to ensure proper scope of records.
```
I would think that USING would be an equality check as the field values have to match for the join.
